deploy/install/02-components/03-ui.yaml

```diff
@@ -19,8 +19,6 @@ spec:
       - name: longhorn-ui
         image: longhornio/longhorn-ui:master-head
         imagePullPolicy: IfNotPresent
-        securityContext:
-          runAsUser: 0
```

Comment:
Because right now our images have the root user, we don't need this actually. But I would prefer to keep it on purpose to notice we need the root for the current components. WDYT?

Reply:
Ok, keep this